app/src/app/navtitle/navtitle.component.sass

```diff
@@ -36,7 +36,7 @@ h2
   position: relative
   top: 3px
 
-  @media (max-width: $breakpoint-tablet)
+//  @media (max-width: $breakpoint-tablet)
     font-size: 9pt
     font-weight: 800
```

Comment:
Why have you changed that? Does not look better now.
![image](https://user-images.githubusercontent.com/1835665/121695893-cd82b800-cacb-11eb-8536-eee3af0037b7.png)